src/store/aggregator/sagas/withdrawTokens.test.ts

```diff
@@ -204,6 +208,14 @@ describe('withdrawTokens', () => {
   });
 
   describe('on RSK only', () => {
+    const txDetails = getTxDetails({
+      isCrossChain: undefined,
```

Comment:
Couldn't it be only a boolean value?

Reply:
yes, it's kinda tricky, but our generic table component accepts strings, not booleans, so the type has to be:
`  isCrossChain?: string | undefined;`

In 'truthy' cases we pass hard coded string `'true'`, but it could be any valid string.